docs/guides/getting-started/setup/vite.mdx

```diff
@@ -78,7 +72,7 @@ export default defineConfig({
   clearScreen: false,
   // Tauri expects a fixed port, fail if that port is not available
   server: {
-    port: 3000,
```

Comment:
We can just omit the port here, as it's the default anyway. Important is only the `strictPort: true` part

Reply:
updated